### ISSUE TITLE
🐛 Critical failure when running `PwBandStructureWorkChain` 

### ISSUE THREAD

**Issue by mbercx** [aiidateam/aiida-tutorials#285]
When trying to run the `PwBandStructureWorkChain` in the Quantum ESPRESSO part of the tutorial, I run into an error related to the `smooth g-vectors` in the first `PwCalculation`:

```
     Dense  grid:    69239 G-vectors     FFT dimensions: (  54,  54,  54)

     Smooth grid:    24399 G-vectors     FFT dimensions: (  36,  36,  36)

     Estimated max dynamical RAM per process >      72.28Mb

 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
     Error in routine ggen (6):
     smooth g-vectors missing !
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

     stopping ...
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
```

This issue is definitely reproducible, I've run into it several times. Try running an AiiDAlab cluster from @csadorf and try running the `demo_bands.py` script.

Will dig some more into what's causing this now, but if anyone else is crazy enough to be working on a Sunday evening (*cough* @giovannipizzi *cough* 🙃) and know what's causing this, let me know!

**Comment by giovannipizzi** [comment]
Is this in the final SCF of a vc-relax? Are you starting with an initial cell *very far away* from the final one? can you post the full input and output of that calc?

**Comment by mbercx** [comment]
> Is this in the final SCF of a vc-relax? 

Yes, this is in the `vc-relax` `PwCalculation`, when it should start the final scf, i.e. after:

```
A final scf calculation at the relaxed structure.
```

> Are you starting with an initial cell _very far away_ from the final one? 

I would think not. The structure is [the conventional unit cell from the COD](http://crystallography.net/cod/9008565.html). (as a sidenote, the structure there doesn't look quite right... 🤔 )

> can you post the full input and output of that calc?

* [aiida.in](https://github.com/aiidateam/aiida-tutorials/files/5615464/aiida.in.txt)
* [aiida.out](https://github.com/aiidateam/aiida-tutorials/files/5615468/aiida.out.txt)

Going to try with another structure from the COD while I dig for the issue.
EDIT: ➡️  [ Si structure from Materials project](https://materialsproject.org/materials/mp-149/#) works fine, so will simply change the structure...

**Comment by giovannipizzi** [comment]
Ok! good news. In any case, avoid using the conventional cell with 8 atoms, use the primitive with 2, it's going to be an order of magnitude faster and you don't get a lot of band refolding that might be confusing at first

**Comment by mbercx** [comment]
Closed by #290
